lib/active_model/serializer/adapter/json_api/api_objects.rb

```diff
@@ -0,0 +1,16 @@
+require 'active_model/serializer/adapter/json_api/api_objects/link'
+require 'active_model/serializer/adapter/json_api/api_objects/resource'
+require 'active_model/serializer/adapter/json_api/api_objects/resource_identifier'
+require 'active_model/serializer/adapter/json_api/api_objects/relationship'
+require 'active_model/serializer/adapter/json_api/api_objects/json_api_object'
+
+module ActiveModel
+  class Serializer
+    module Adapter
+      class JsonApi
+        module ApiObjects
```

Comment:
I think these should be autoloaded, but don't feel strongly about it

I also like to put the requires in the module body, but am not sure if we've picked a pattern to use or model. Usually we reference ActiveJob